frontend/src/metabase/dashboard/analytics.ts

```diff
@@ -12,10 +12,25 @@ export const trackAutoApplyFiltersDisabled = (dashboardId: DashboardId) => {
   });
 };
 
-export const trackExportDashboardToPDF = (dashboardId: DashboardId) => {
+type DashboardAccessedVia =
```

Comment:
This is the same type as `ResourceAccessedVia`, can we reuse it?

https://github.com/metabase/metabase/blob/292c6144036a86a53917f43e8e719a0140f92e69/frontend/src/metabase/redux/downloads.ts#L43-L48